airflow/www/templates/airflow/dag.html

```diff
@@ -272,14 +276,17 @@ <h4 class="modal-title" id="taskInstanceModalLabel">
             </div>
           {% endif %}
           <h4>Task Actions</h4>
-          <form method="POST" data-action="{{ url_for('Airflow.run') }}">
+          <form method="POST" data-action="{{ url_for('Airflow.run') }}" id="run_action">
             <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
             <input type="hidden" name="dag_id" value="{{ dag.dag_id }}">
             <input type="hidden" name="task_id">
             <input type="hidden" name="dag_run_id">
             <input type="hidden" name="map_index">
             <input type="hidden" name="origin" value="{{ request.base_url }}">
             <div class="row">
+              <span class="col-xs-12 col-sm-9 text-danger" style="font-size: 12px">
+                {{ "Only works with the Celery, CeleryKubernetes or Kubernetes executors, sorry" if not k8s_or_k8scelery_executor else "" }}
```

Comment:
Keep this consistent with the other instance in React
```suggestion
                {{ "Only works with the Celery, CeleryKubernetes or Kubernetes executors" if not k8s_or_k8scelery_executor else "" }}
```